screens/MainScreen.tsx

```diff
@@ -1,346 +1,404 @@
+  //   right: '0%',
+  //   top: '0%',
+  //   height: '100%',
+  //   bottom: '0%',
+  //   position: 'absolute',
+  //   width: '100%',
+  // },
+  // brandColorBg: {
+  //   left: '0%',
+  //   right: '0%',
+  //   top: '0%',
+  //   height: '100%',
+  //   bottom: '0%',
+  //   position: 'absolute',
+  //   width: '100%',
+  // },
+  // livingBackgroundIcon: {
+  //   left: '0%',
+  //   bottom: '0%',
+  //   right: '0%',
+  //   top: '0%',
+  //   height: '100%',
+  //   width: '100%',
+  // },
+  // bg: {
+  //   top: 0,
+  //   height: 812,
+  // },
+  // homeIndicator: {
+  //   marginLeft: -66.5,
+  //   bottom: 8,
+  //   left: '50%',
+  //   borderRadius: Border.br_81xl,
+  //   width: 134,
+  //   height: 5,
+  // },
+  // tabbarHandleIos: {
+  //   height: '4.19%',
+  //   top: '95.81%',
+  //   left: '0%',
+  //   right: '0%',
+  //   position: 'absolute',
+  //   width: '100%',
+  // },
+  // border: {
+  //   width: '90.43%',
+  //   right: '9.57%',
+  //   borderRadius: 3,
+  //   borderStyle: 'solid',
+  //   borderColor: '#fff',
+  //   borderWidth: 1,
+  //   opacity: 0.35,
+  //   left: '0%',
+  //   top: '0%',
+  //   height: '100%',
+  //   bottom: '0%',
+  //   position: 'absolute',
+  // },
+  // capIcon: {
+  //   height: '35.29%',
+  //   width: '5.46%',
+  //   top: '32.35%',
+  //   bottom: '32.35%',
+  //   left: '94.54%',
+  //   opacity: 0.4,
+  //   right: '0%',
+  // },
+  // capacity: {
+  //   height: '64.71%',
+  //   width: '73.99%',
+  //   top: '17.65%',
+  //   right: '17.79%',
+  //   bottom: '17.65%',
+  //   left: '8.22%',
+  //   borderRadius: 1,
+  // },
+  // battery: {
+  //   height: '25.76%',
+  //   width: '6.49%',
+  //   top: '39.39%',
+  //   right: '3.82%',
+  //   bottom: '34.85%',
+  //   left: '89.69%',
+  //   position: 'absolute',
+  // },
+  // wifiIcon: {
+  //   width: 15,
+  //   height: 11,
+  // },
+  // lte: {
+  //   top: 17,
+  //   left: 312,
+  //   fontSize: 11,
+  //   letterSpacing: -1,
+  //   fontWeight: '500',
+  //   fontFamily: FontFamily.sFProTextMedium,
+  //   width: 25,
+  //   height: 12,
+  //   display: 'none',
+  // },
+  // cellularIcon: {
+  //   height: '24.24%',
+  //   width: '4.53%',
+  //   top: '40.15%',
+  //   right: '17.07%',
+  //   bottom: '35.61%',
+  //   left: '78.4%',
+  // },
+  // time: {
+  //   marginTop: -3.5,
+  //   top: '50%',
+  //   letterSpacing: 0,
+  //   fontWeight: '600',
+  //   fontFamily: FontFamily.sFProTextSemibold,
+  //   fontSize: FontSize.size_mini,
+  //   left: '0%',
+  //   width: '100%',
+  // },
+  // barsStatusBarIphoneT: {
+  //   height: '47.73%',
+  //   width: '14.4%',
+  //   top: '15.91%',
+  //   right: '80%',
+  //   bottom: '36.36%',
+  //   left: '5.6%',
+  //   position: 'absolute',
+  // },
+  // barsStatusBarIphoneL: {
+  //   height: '5.42%',
+  //   bottom: '94.58%',
+  //   left: '0%',
+  //   right: '0%',
+  //   top: '0%',
+  //   position: 'absolute',
+  //   width: '100%',
+  // },
```

Comment:
Please Remove the commented code, remember not to push any unused code to the PR